sphinxcontrib/confluencebuilder/translator/storage.py

```diff
@@ -742,13 +749,22 @@ def depart_tbody(self, node):
     def visit_row(self, node):
         self.body.append(self._start_tag(node, 'tr', suffix=self.nl))
         self.context.append(self._end_tag(node))
+        # for stub-columns implmentation
+        # see https://svn.code.sf.net/p/docutils/code/trunk/docutils/docutils/writers/_html_base.py
+        node.column = 0
 
     def depart_row(self, node):
         self.body.append(self.context.pop()) # tr
 
     def visit_entry(self, node):
         if self._thead_context[-1]:
             target_tag = 'th'
+        elif self._stub_columns_experimental and \
+                node.parent.parent.parent.stubs[node.parent.column]:
+            # for stub-columns implementation
+            # see https://svn.code.sf.net/p/docutils/code/trunk/docutils/docutils/writers/_html_base.py
+            target_tag = 'th'
+            node.parent.column += 1
```

Comment:
The `node.parent.column` increment here should be moved outside of the if-condition (as done in `_html_base.py`), otherwise it will only increment a column count for the stub cases. This could provide flexibility later if anything else needs to reference the column count.